### ISSUE TITLE
Monitoring servo outputs while in mix screens

### ISSUE THREAD

**Issue by Eurylokhos** [FrSky-OS/discussion#49]
If there is a way to do this, I'm not seeing it. In the mix screens, I do not see a way to monitor what the channel outputs are doing. This is an important feature when fine tuning mixes. A button on the touchscreen, or the ability to hold down a single button to jump to a servo monitor screen is vital with all of these available mixes for setup and tuning.

**Comment by ask21flyer** [comment]
if i understand correctly what you are looking for,
select model - on first page - second heading down- servo

**Comment by Eurylokhos** [comment]
Yes, it is there if you are going to jump out of your menu then go to another one, scroll down, select it, see the results, then go through it all again to get back to where you were to fine tune things going back and forth multiple times. I would like a way to quickly jump to the monitor with a single press from a mix screen to make it quick to fine tune things. Many radios have this, it makes programming far easier. That is why I said "in the mix screens". It simplifies complex programming.

**Comment by ask21flyer** [comment]
good point you make

**Comment by AloftHobbies** [comment]
Yes, this would be a very nice addition. Perhaps the "page down" button could jump to servo monitor screen, and push "page up" to jump back to the mix screen.
